### DISCUSSION TITLE
Execute print command only at Debug

### DISCUSSION THREAD

**Discussion by vincius0000** [enrique-lozano/Monekin#229] [Ideas / New features]
Hello!

Currently IDE gives lot of warning of `Print` being used in Production. I suggest adding following file to utils folder and replacing `print` with `Logger.printDebug` everywhere

```dart
// logger.dart
import 'package:flutter/foundation.dart';

class Logger {
  static void printDebug(String message) {
    if (kDebugMode) {
      print(message);
    }
  }
}
```

Unless there's a specific purpose the print is being used in Prod build.

**Comment by enrique-lozano** [comment]
Good idea! I will need to see how other apps manage this in terms of code. PRs appreciated

**Reply by vincius0000** [reply]
@enrique-lozano https://github.com/enrique-lozano/Monekin/pull/249
